projects/components/src/table/table.component.scss

```diff
@@ -88,34 +88,43 @@ $data-row-border: 1px solid $gray-2;
 }
 
 .header-cell {
-  display: flex;
```

Comment:
Is it possible to keep these styles? Just trying to have minimal changes to the code.

Reply:
Visual Styles are almost the same, nothing major change. It's just it is moved around because of the decoupling of the column divider and resize handler in the template.